src/reducers/org.js

```diff
@@ -1033,73 +1033,80 @@ function updatePlanningItemsWithRepeaters(
-        propertyListItems.some(item => item.get('property') === 'LAST_REPEAT')
-          ? propertyListItems.map(item =>
-              item.get('property') === 'LAST_REPEAT'
-                ? item.set('value', fromJS(newLastRepeatValue))
-                : item
-            )
-          : propertyListItems.push(
-              fromJS({
-                property: 'LAST_REPEAT',
-                value: newLastRepeatValue,
-                id: generateId(),
-              })
-            )
-      );
-      state = state.updateIn(['headers', headerIndex], header => {
-        let rawDescription = header.get('rawDescription');
-        if (rawDescription.startsWith('\n')) {
-          rawDescription = rawDescription.slice(1);
-        }
-        rawDescription =
-          `\n- State "${newTodoState}"       from "${currentTodoState}"       ${renderAsText(
-            fromJS(lastRepeatTimestamp)
-          )}\n` + rawDescription;
-        return header
-          .set('rawDescription', rawDescription)
-          .set('description', parseRawText(rawDescription));
-      });
-    }
+  });
+  state = state.setIn(
+    ['headers', headerIndex, 'titleLine', 'todoKeyword'],
+    currentTodoSet.get('keywords').first()
+  );
+  if (noLogRepeatEnabledP({ state, headerIndex })) {
+    const lastRepeatTimestamp = getCurrentTimestamp({ isActive: false, withStartTime: true });
+    const newLastRepeatValue = [
+      {
+        type: 'timestamp',
+        id: generateId(),
+        firstTimestamp: lastRepeatTimestamp,
+        secondTimestamp: null,
+      },
+    ];
+
+    state = state.updateIn(['headers', headerIndex, 'propertyListItems'], propertyListItems =>
+      propertyListItems.some(item => item.get('property') === 'LAST_REPEAT')
+        ? propertyListItems.map(item =>
+            item.get('property') === 'LAST_REPEAT'
+              ? item.set('value', fromJS(newLastRepeatValue))
+              : item
+          )
+        : propertyListItems.push(
+            fromJS({
+              property: 'LAST_REPEAT',
+              value: newLastRepeatValue,
+              id: generateId(),
+            })
+          )
+    );
+    state = state.updateIn(['headers', headerIndex], header => {
+      let rawDescription = header.get('rawDescription');
+      if (rawDescription.startsWith('\n')) {
+        rawDescription = rawDescription.slice(1);
+      }
+      rawDescription =
+        `\n- State "${newTodoState}"       from "${currentTodoState}"       ${renderAsText(
+          fromJS(lastRepeatTimestamp)
+        )}\n` + rawDescription;
+      return header
+        .set('rawDescription', rawDescription)
+        .set('description', parseRawText(rawDescription));
+    });
   }
   return state;
 }
 
+/**
+ * Is the `nologrepeat` feature enabled for this buffer?
+ * More info:
+ * https://www.gnu.org/software/emacs/manual/html_node/org/Repeated-tasks.html
+ */
+export function noLogRepeatEnabledP({ state, headerIndex }) {
```

Comment:
Should this be called `logRepeatEnabledP` instead, since it returns `true` if it should repeat and `false` if `nologrepeat` is set?

Reply:
Good question! I was actually thinking about this name for a moment when I had the same idea as you^^

I'm not super happy about the name, because it's quite long, but here's my thought process: The native Org feature is called `nologrepeat`. Hence it makes sense to  call the feature flag in organice by the same name and not invert the native feature flags name. The feature flag is enabled when "nologrepeat" is 'enabled' or 'set'. 

Atm it's kinda hard to read and if I would do it from scratch, I'd call the feature "logrepeatenabled", but Org mode has prior art here(;

On the other hand, having you trip up over the same naming issue that I was already thinking about is not a great argument for the name. So I'm open for a change if after reading this you have a strong fealing it woud be better the other way around!

Reply:
What if we keep the name, but invert what it returns, so when nologrepeat is set it returns true? That way we keep the name, but it reads a bit more naturally; e.g. the check for logging the repeat information would be `if (!noLogRepeatEnabledP(...)) { /* log repeat /* }`

Reply:
Oh, my, of course! It's a little late for me here, I apologize. 

I should take a break, maybe(;

Reply:
😄 Can't hurt! I can make that change & push it up

Reply:
Sweet!

Also thanks for adding the test for detecting via properties. I was just starting to type that one in when I got the notification about your commit(;

I'll happily pull the changes in when you ping me.